src/styles/layout.scss

```diff
@@ -13,20 +13,21 @@
     background-size: 900px 900px;
     background-repeat: no-repeat;
     background-position: top -200px right -200px;
-    @include darkMode {
-      background-image: radial-gradient(
-        hsla(176, 68%, 41%, 0.11) 0,
-        var(--amplify-colors-background-primary) 70%,
-        var(--amplify-colors-background-primary) 100%
-      );
-    }
+
     &.layout-wrapper--gen1 {
       background-image: radial-gradient(
         hsla(176, 68%, 41%, 0.21) 0,
         #fff 70%,
         #fff 100%
       );
     }
+    @include darkMode {
+      background-image: radial-gradient(
+        hsla(176, 68%, 41%, 0.11) 0,
+        var(--amplify-colors-background-primary) 70%,
+        var(--amplify-colors-background-primary) 100%
+      );
+    }
```

Comment:
I noticed that on dark mode the spaceship bg had a white bg. I think it's because the `layout-wrapper--gen1` class was overwriting the darkmode style so I swapped their positions to fix this